buildpipeline/pipeline.json

```diff
@@ -150,7 +150,8 @@
         "TreatWarningsAsErrors": "false"
       },
       "BuildParameters": {
-        "PB_ConfigurationGroup": "Release"
+        "PB_ConfigurationGroup": "Release",
+        "PB_BuildArguments": "-stripSymbols"
```

Comment:
I could change the help message from:

> stripSymbols - Optional argument to strip native symbols during the build.

to

> stripSymbols - Optional argument to strip native symbols and put them into separate symbols files during the build. If this is not passed, symbol information is left in the native output files. Regardless of this option, when NuGet packages are created, symbol packages are created if any symbols files exist.

I can also put this in config.json for `run -?` output.